forge-gui/res/cardsfolder/upcoming/viral_spawning.txt

```diff
@@ -0,0 +1,9 @@
+Name:Viral Spawning
+ManaCost:2 G
+Types:Sorcery
+A:SP$ Token | TokenAmount$ 1 | TokenScript$ g_3_3_phyrexian_beast_toxic | TokenOwner$ You | SpellDescription$ Create a 3/3 green Phyrexian Beast creature token with toxic 1. (Players dealt combat damage by this creature also get a poison counter.)
+S:Mode$ Continuous | Affected$ Card.Self | CheckSVar$ X | SVarCompare$ GE3 | EffectZone$ Graveyard,Stack | AffectedZone$ Graveyard,Stack | AddKeyword$ Flashback:2 G | Description$ Corrupted — As long as an opponent has three or more poison counters and CARDNAME is in your graveyard, it has flashback {2}{G}. (You may cast this card from your graveyard for its flashback cost. Then exile it.)
```

Comment:
i think it was similar to the Blitz one
that static abilities that let you cast it continues to apply while it is on the stack

so we probably need to tweak it in the engine somehow

Reply:
Sounds like this one:
> > 400.7g. If an effect grants a nonland card an ability that allows it to be cast, that ability will continue to apply to the new object that card became after it moved to the stack as a result of being cast this way.

The tricky part will probably be with "cast this way" making sure it only keeps the right one, in case it gained a second flashback

I tried adding `setKeyword` in GameActionUtil when it builds the flashback SA but that didn't help